code/common/api.q

```diff
@@ -72,6 +72,21 @@ search:{[s;c]
 // search function
 s:.api.search[;0b]
 
+// input list of namespaces for exportconfig
+KEYS:` sv'`,'key[`]except`$'.Q.an;
```

Comment:
Can you give this a better name like "torqnamespaces" or something.  Keys is a bit vague